homeassistant/components/mqtt/climate.py

```diff
@@ -390,7 +390,7 @@ def add_subscription(topics, topic, msg_callback):
 
         def render_template(msg, template_name):
             template = self._value_templates[template_name]
-            return template(msg.payload)
+            return template(msg.payload, variables={"entity_id": self.entity_id})
```

Comment:
You need to update the default lambda when there is no template, see the failing CI test.

Reply:
Added in 2c70b7a87c.